eng/feeds/azure-canonical-scaffolding.json

```diff
@@ -0,0 +1,66 @@
+{
+  "azure-arm-canonical": {
+    "title": "(rest-api-spec repo) Azure Resource Manager Service Project with Canonical Swagger",
+    "description": "Create a project with canonical swagger in rest-api-spec repo, representing an Azure service ARM API",
```

Comment:
`Create standard ARM API project with an additional canonical swagger emitter in rest-api-spec repo`

Reply:
Just noticed it is separate scaffolding json. Above not neccessary.